modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -32,77 +32,154 @@ on {rh-openstack-first}
 
-* Load balance the application ports, 443 and 80, between all the compute nodes.
+* For a backend IP address, ensure that an IP address for an {product-title} control plane node does not change during the lifetime of the external load balancer. You can achieve by completing one of the following actions:
+** Assign a static IP address to each control plane node.
+** Configure each node to receive the same IP address from the DHCP every time the node requests a DHCP lease. Depending on the vendor, the DHCP lease might be in the form of an IP reservation or a static DHCP assignment. 
 
-* Load balance the API port, 6443, between each of the control plane nodes.
+* For the Ingress Controller backend service, ensure you manually define each node that runs the Ingress Controller in the external load balancer. For example, if the Ingress Controller moves to an undefined node, an connection outage can occur. 
 
-* On your load balancer, port 22623, which is used to serve ignition startup configurations to nodes, is not exposed outside of the cluster.
+.OpenShift API prerequisites
 
-* Your load balancer can access the required ports on each node in your cluster. You can ensure this level of access by completing the following actions:
-** The API load balancer can access ports 22623 and 6443 on the control plane nodes.
-** The ingress load balancer can access ports 443 and 80 on the nodes where the ingress pods are located.
+* You defined a front-end IP address.
+* You ensured that on your load balancer, TCP ports 6443, 443, 22623, and 80 are exposed on the front-end IP address. Check the following items:
+** Port 6443 provides access to the OpenShift API service.
+** Port 22623 can provide ignition startup configurations to nodes. 
+* The front-end IP address and port 6443 are reachable by all users of your system with a location external to your {product-title} cluster.
+* The front-end IP address and port 22623 are reachable only by {product-title} nodes.
+* The load balancer backend can communicate with {product-title} control plane nodes on port 6443 and 22623.
 
-ifdef::vsphere[]
-* Optional: If you are using multiple networks, you can create targets for every IP address in the network that can host nodes. This configuration can reduce the maintenance overhead of your cluster.
-endif::vsphere[]
+.Ingress Controller prerequisites
+
+* You defined a front-end IP address.
+* You ensured that on your load balancer, TCP ports 443 and 80 are exposed on the front-end IP address.
+* The front-end IP address, port 80 and port 443 are be reachable by all users of your system with a location external to your {product-title} cluster. 
+* The front-end IP address, port 80 and port 443 are reachable to all nodes that operate in your {product-title} cluster.
+* The load balancer backend can communicate with {product-title} nodes that run the Ingress Controller on ports 80, 443, and 1936.
 
 .Procedure
```

Comment:
Instead of procedure, maybe "Configuration Example?"

Reply:
Hi Clark. The procedure title is essential for a procedure-type module. If I change the module to a concept type, I would need to remove a lot of the headings and remove the action steps "you must, and so on". WDYT?

Reply:
Rereading through, I think Procedure is OK.  

What's confusing on reading this through is the first step "Configure Health Check Resources for your backend services.

I think if we move Line 89 - 123 into a section _before_ the Procedure.  Called "Health Check URL Specifications"

Reply:
"Most load balancers allow setting health check URLs to determine if a services is available or not.  OpenShift provides these for the OpenShift API, Machine Configuration API, and Ingress Controller.

Reply:
Hi @clarkhale . Just wondering about "OpenShift provides these for the OpenShift API, Machine Configuration API, and Ingress Controller." . Does OCP provide load balancers, health checks, or services for the listed APIs?


OpenShift API, Machine Configuration API, and Ingress Controller = services?

Reply:
> Does OCP provide load balancers, health checks, or services for the listed APIs?
Health checks

> OpenShift API, Machine Configuration API, and Ingress Controller = backend services?

Collectively, yes, in context of the load balancer.

Reply:
Thanks, Clark. Updated.